h/browser/chrome/test/sidebar-injector-test.js

```diff
@@ -7,10 +7,30 @@ describe('SidebarInjector', function () {
   var fakeChromeTabs;
   var fakeFileAccess;
 
+  // the content type that the detection script injected into
+  // the page should report ('HTML' or 'PDF')
+  var contentType;
+  // the return value from the content script which checks whether
+  // the sidebar has already been injected into the page
+  var isAlreadyInjected;
```

Comment:
I'd prefer to avoid these globals, but I suppose to do so you'd have to refactor existing tests?